lib/wallet/api/http/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -2361,8 +2361,10 @@ postTransactionFeeOld ctx@ApiLayer{..} (ApiT walletId) body = do
     protocolParameters <- liftIO $ currentProtocolParameters netLayer
     era <- liftIO $ NW.currentNodeEra netLayer
     AnyRecentEra (recentEra :: WriteTx.RecentEra era) <- guardIsRecentEra era
+    let mTTL = body ^? #timeToLive . traverse . #getQuantity
     withWorkerCtx ctx walletId liftE liftE $ \workerCtx -> do
         let db = workerCtx ^. dbLayer
+        ttl <- liftIO $ W.transactionExpirySlot (timeInterpreter netLayer) mTTL
```

Comment:
On the fence about this one. It seems like it should be included, but it wasn't before. The time-slot conversion shouldn't fail any more than it would fail during normal tx creation (I think never)... so I think there shouldn't be any downside to this.